src/lib.rs

```diff
@@ -389,13 +389,13 @@ impl<S: Storage + Send + Sync + 'static> SelfEncryptor<S> {
         if file_size <= MAX_IN_MEMORY_SIZE as u64 {
             sequencer = Sequencer::as_vector();
         } else {
-            sequencer = Sequencer::as_mmap();
+            sequencer = Sequencer::as_mmap().unwrap();
```

Comment:
Yes, my change here is just a first step towards proper error management. Maybe in another PR...